### ISSUE TITLE
Ingestion: unix pipe writer

### ISSUE THREAD

**Issue by ire-and-curses** [stellar/go#1394]
A writer for the ingestion system that emits to a Unix pipe,  file or socket would provide a convenient way to escape from Go for secondary processing in a language of the user's choice. This would greatly expand the versatility of the pipeline for use by non-Go developers, without adding any additional libraries or middleware.

**Comment by bartekn** [comment]
This is actually what captive core does now (stream `xdr.LedgerCloseMeta`). I wonder if we should close this.

**Comment by ire-and-curses** [comment]
Yeah maybe. Then again that stream still needs to be decoded so it's not like you can just pipe it to grep or something.

**Comment by ire-and-curses** [comment]
If it was possible and documented with an example then this can be closed. See #2815.

**Comment by 2opremio** [comment]
I _think_ we can close this. @ire-and-curses please reopen if you disagree.

**Comment by ire-and-curses** [comment]
What I want is a worked example of how to use the ingestion library to dump to stdout. Basically, how do I make a tool that can be composed with other unix command line tools. Do we have that?
